packages/expect/src/index.ts

```diff
@@ -137,8 +137,8 @@ const expect: any = (actual: any, ...rest: Array<any>) => {
   return expectation;
 };
 
-const getMessage = (message?: () => string) =>
-  (message && message()) ||
```

Comment:
should check if it is a function.

`typeof message === 'function' ? message() : message` or something